app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+    before_action :authenticate_user!, except: [:index]
+    # before_action :find_post!
+
+    def new
+        
+    end
+    
+
+    def index
+        @comments = @post.comments.order(created_at: :desc)
+    end
+
+    def create
+        @post = Post.find(params[:comment][:post_id])
+        @comment = @post.comments.build(comment_params)
+        @comment.user = current_user
+        if @comment.save
+            respond_to do |format|
+                format.js 
+            end
+        end
+    end
+
+    def destroy
```

Comment:
[OPTIONAL]

You have the destroy action in here but you don't use it in your views.